ddl/scheduler.go

```diff
@@ -41,6 +42,7 @@ type backfillSchedulerHandle struct {
 	jc          *JobContext
 	eleTypeKey  []byte
 	totalRowCnt int64
+	isPartition bool
```

Comment:
Is this necessary? `backfillSchedulerHandle.ptbl` should be able to get the info.